lib/elixir/test/elixir/kernel/expansion_test.exs

```diff
@@ -1201,16 +1201,18 @@ defmodule Kernel.ExpansionTest do
     end
 
     test "expects one argument in clauses" do
-      assert_raise CompileError, ~r"expected one arg for :do clauses \(->\) in \"cond\"", fn ->
-        code =
-          quote do
-            cond do
-              _, _ -> :ok
-            end
-          end
+      assert_raise CompileError,
+                   ~r"expected one argument for :do clauses \(->\) in \"cond\"",
+                   fn ->
+                     code =
+                       quote do
+                         cond do
+                           _, _ -> :ok
+                         end
+                       end
 
-        expand(code)
-      end
+                     expand(code)
+                   end
```

Comment:
this whole chunk is changed by the formatter. the only change is in the error message